scalameter-core/src/main/java/org/scalameter/execution/invocation/instrumentation/MethodInvocationCounter.java

```diff
@@ -0,0 +1,74 @@
+package org.scalameter.execution.invocation.instrumentation;
+
+import java.util.concurrent.atomic.AtomicLongArray;
+
+
+/** Internal API
+ *
+ *  Stores method invocations in an AtomicLongArray.
+ *
+ *  Note that this class is a Java class because we want method [[methodCalled]] to be static,
+ *  which makes the bytecode instrumentation easier and more compact.
+ */
+public class MethodInvocationCounter {
```

Comment:
One more detail - can you use 2 space indentation throughout Java files as well.